.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Please add `.vscode/` and `settings.json` to the `.gitignore` file. You'll need to delete the settings file locally as well when you push.